package user;
option cc_generic_services = true;
option java_outer_classname = "LoginPro";

message LoginReq {
  required int64 uid = 1;
  required string userName = 2;
  repeated string userPasswd = 3;
}

message LoginRsp {
  required int64 uid = 1;
  required int32 status = 2;
  required string username = 3;
}

message LoginOutReq {
  required int64 uid = 1;
  required string userName = 2;
}

message LoginOutRsp {
  required int64 uid = 1;
  required int32 status = 2;
  required string username = 3;
}

service LoginService {
  rpc Login (LoginReq) returns (LoginRsp);
}

service LoginOutService {
  rpc LoginOut (LoginOutReq) returns (LoginOutRsp);
}

